templates/_macros/hero.jinja

```diff
@@ -0,0 +1,127 @@
+# Params
+# title_text: Hero title text (required)
+# subtitle_text: Hero subtitle text
+# layout: layout of hero section. Options are '50/50', '75/25', '25/75', 'fallback'
+# is_split_on_tablet: whether the layout is split on tablet. Applies "-on-medium" to row class in 50/50 and 25/75 layouts. If false, the layout is stacked on tablet.
+# is_split_on_desktop: whether the layout is split on desktop. Applies "-on-large" to row class in 50/50 and 25/75 layouts. If false, the layout is stacked on desktop, provided is_split_on_tablet is true.
+# is_image_full_cover: whether the image should span full width of the hero. Only supported on 50/50 layout
```

Comment:
Full cover is also available on 25-75 signpost, but it is [mandatory](https://www.figma.com/design/Hll6ZRvHyTTGBZZSoRd8ei/Higher-order-components-(HOCs)?node-id=693-12631&m=dev) there for heros that have images

So, in practice, this flag is only used in 50-50 layout; you're right that it is a bit odd to have the flag in this case. I came to the same conclusion that it's better to handle the variation in this macro with a flag than create a new macro or different layout option

Reply:
This was moved to a `50/50-full-width-image` layout for now. `layout='25-75'` and `layout='50/50-full-width-image'` will both set `is_image_full_width=true`, but it will not be exposed to the caller directly; it will be set based on `layout` choice.